.storybook/preview.js

```diff
@@ -1,8 +1,8 @@
 import '../src/customElements/index.ts';
 import { addDecorator } from '@storybook/react';
-import results from '../.jest-test-results.json';
-import { withTests } from '@storybook/addon-jest';
+// // import results from '../.jest-test-results.json';
```

Comment:
Remove comments?

Reply:
we might reimplement jest at a later point